.gitignore

```diff
@@ -36,3 +36,6 @@ docker-compose-run-local.yml
 main.log
 server.log
 tests.test
+
+# Forge docs
+contracts/docs/
```

Comment:
I don't think this should be ignored

Reply:
Removed to decouple this change from https://github.com/ava-labs/teleporter/pull/87. We don't check in the docs in that PR since the docs themselves contain the commit from which they were generated. This makes it difficult to keep them up to date in an automated way.